### PR TITLE
Added Fullscreen and Unfullscreen buttons in Tankop Activity

### PR DESCRIPTION
**Pull request by s-mag**
@llaske Made a new PR

### CODE REVIEW

**Comment by llaske** [comment]
It's nice but the resize should be call when fullscreen is called else the board is not fully used.

**Comment by s-mag** [comment]
> It's nice but the resize should be call when fullscreen is called else the board is not fully used.

@llaske that gives a error when called..
see here
Error2

**Comment by llaske** [comment]
Should be probably called only when on the play screen.

**Comment by s-mag** [comment]
> Should be probably called only when on the play screen.

If I call it in play mode it does not do anything..
Can you suggest any another way

**Comment by s-mag** [comment]
Added back resize function

**Comment by s-mag** [comment]
@llaske 
Will it be fine if this PR fixes the other issues and not the responsiveness.
I will fix the responsiveness in other PR.
Can we do this???

File: activities/TankOp.activity/js/activity.js [hunk 1]
```diff
@@ -38,30 +38,19 @@ define(["sugar-web/activity/activity"], function (activity) {
...
-						} 
-					}
-
-					function zoomout(){
-						var x=document.getElementsByClassName("body");
-						for (var i = 0; i < x.length; i++) {
-							x[i].style.bottom="0px";
-						} 
-					}
+					play= new TankOp.Play();
```
Comment (llaske) [inline activities/TankOp.activity/js/activity.js]: Not a good idea to create a new Play object. Play object is the play screen so it should be created only when the play screen is displayed.

File: activities/TankOp.activity/js/activity.js [hunk 2]
```diff
@@ -38,30 +38,19 @@ define(["sugar-web/activity/activity"], function (activity) {
...
 					// Full screen
 					document.getElementById("fullscreen-button").addEventListener('click', function() {
 						document.getElementById("main-toolbar").style.visibility = "hidden";
 						document.getElementById("unfullscreen-button").style.visibility = "visible";
-						document.getElementById("unfullscreen-button").style.top = "40px";
-						zoomin();
+						if (app.init())
```
Comment (llaske) [inline activities/TankOp.activity/js/activity.js]: Not a good idea to call init, it should be called once at the activity initialization.

**Comment by s-mag** [comment]
@llaske can you suggest any another way to make it work...

File: activities/TankOp.activity/js/activity.js (hunk 2 above)
Comment (s-mag) [reply activities/TankOp.activity/js/activity.js]: Then how should I check the play mode

**Comment by llaske** [comment]
Look like the `TankOp.Play` object is already stored in a global variable `play` [here](https://github.com/llaske/sugarizer/blob/dev/activities/TankOp.activity/play.js#L59).
May be you could try to use it.

**Comment by s-mag** [comment]
@llaske Fixed the errors and made changes...
Can you help me out to resize the screen as calling it does not resize it and at the same time no error is passed which resolves the changes requested..

**Comment by llaske** [comment]
It's nice but there is still an error. 

```
play.js:173 Uncaught TypeError: Cannot read property 'id' of null
    at e.TankOp.Play.resize() [as resize] (play.js:173)
    at HTMLButtonElement.<anonymous> (activity.js:65)
TankOp.Play.resize() @ play.js:173
(anonymous) @ activity.js:65
```

Steps to reproduce:
- Click on fullscreen
- Click on start
- Click on home
- Click on unfullscreen

**Comment by s-mag** [comment]
> It's nice but there is still an error.
> 
> ```
> play.js:173 Uncaught TypeError: Cannot read property 'id' of null
>     at e.TankOp.Play.resize() [as resize] (play.js:173)
>     at HTMLButtonElement.<anonymous> (activity.js:65)
> TankOp.Play.resize() @ play.js:173
> (anonymous) @ activity.js:65
> ```
> 
> Steps to reproduce:
> 
> * Click on fullscreen
> * Click on start
> * Click on home
> * Click on unfullscreen

Will fix the error. Is there anything else to do apart from this ??

**Comment by s-mag** [comment]
@llaske Fixed these errors...

**Comment by s-mag** [comment]
@llaske Is anything to be done for this issue??

**Comment by llaske** [comment]
Just fix the changes requested and it will be okay.

**Comment by s-mag** [comment]
I guess I am not getting what to change. You requested changes but the changes are not shown...

**Pull request merged**